.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to Stockfish
+
+Welcome to the Stockfish project! We're excited that you're interested in contributing. This document outlines the guidelines and steps to follow when making contributions to Stockfish.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Setting up Stockfish](#setting-up-stockfish)
+  - [Building Stockfish](#building-stockfish)
+- [Making Contributions](#making-contributions)
+  - [Reporting Issues](#reporting-issues)
+  - [Submitting Pull Requests](#submitting-pull-requests)
+- [Code Style](#code-style)
+- [Community and Communication](#community-and-communication)
+- [License](#license)
+
+## Getting Started
+
+### Setting up Stockfish
+
+1. Fork this repository.
+2. Clone your forked repository: `git clone https://github.com/your-username/Stockfish.git`
+3. Add the Stockfish repository as a remote: `git remote add upstream https://github.com/official-stockfish/Stockfish.git`
+
+### Building Stockfish
+
+In case you dont have a C++ Compiler installed you can follow the instructions from our wiki [linux](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#linux)/[windows](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#windows)/[macos](https://github.com/official-stockfish/Stockfish/wiki/Compiling-from-source#macos) to install one.
+
+## Making Contributions
+
+### Reporting Issues
+
+If you find a bug, please open an issue on the [issue tracker](https://github.com/official-stockfish/Stockfish/issues). Be sure to include relevant information like your operating system, build environment, and a detailed description of the problem.
+
+_Please note that Stockfishs development isn't focused on adding new features. So any issues regarding missing features will potentially be closed without further discussion._
+
+### Submitting Pull Requests
+
+1. Make sure your forked repository is up to date with the official Stockfish repository: `git pull upstream master`
+2. Create a new branch for your changes: `git checkout -b branch-name`
```

Comment:
what about using the new syntax `git switch -c branch-name>` ?

Reply:
I get this error:
```
    git switch -c my-branch  
    git: 'switch' is not a git command. See 'git --help'.
```

Reply:
git 2.23.0 introduced in 2019 the switch command.

Reply:
changed to `switch`